components/SwitchSurveyExit.client.vue

```diff
@@ -1,17 +1,27 @@
 <template>
+<Modal v-if="surveyContent != null" v-model="showModal">
     <div class="flex flex-col items-center">
-        <PrismicRichText v-if="surveyContent?.data.title" :field="surveyContent.data.title" class="text-3xl mb-6 font-semibold whitespace-pre-wrap" />
-        <PrismicRichText v-if="surveyContent?.data.subtitle" :field="surveyContent.data.subtitle" class="text-lg mb-6 font-semibold whitespace-pre-wrap" />
+        <PrismicRichText v-if="surveyContent?.data?.title" :field="surveyContent.data.title" class="text-3xl mb-6 font-semibold whitespace-pre-wrap" />
+        <PrismicRichText v-if="surveyContent?.data?.subtitle" :field="surveyContent.data.subtitle" class="text-lg mb-6 font-semibold whitespace-pre-wrap" />
         <NuxtLink @click="clickSurvey" to="/impact"
```

Comment:
The link is specified in the prismic document so shouldn't be hardcoded


https://bankgreen.prismic.io/documents~w=Did%20you%20open%20a%20green%20account%20as%20a%20result%20of%20visiting%20our%20website%3F&b=working&c=published&l=en-gb/ZGiGDBAAACMAfL2q/

Reply:
This looks good to me!

Reply:
changed link to get pulled from prismic